concourse/templates.libsonnet

```diff
@@ -657,4 +657,34 @@
         },
       },
     },
+
+  // Sends GQL changes of subgraph to schema registry
```

Comment:
would be good to leave a note here that `get vault` task with 'vault/deploy/graphql/schema-registry' is required for this task.

Reply:
Good point, added, thanks